packages/block-library/src/file/edit.native.js

```diff
@@ -175,27 +191,40 @@ export class FileEdit extends Component {
 	}
 
 	getInspectorControls(
-		{ showDownloadButton, textLinkTarget },
+		{ showDownloadButton, textLinkTarget, href, textLinkHref },
+		media,
 		isUploadInProgress,
 		isUploadFailed
 	) {
+		let linkDestinationOptions = [ { value: href, label: __( 'URL' ) } ];
+		const attachmentPage = media && media.link;
+
+		if ( attachmentPage ) {
+			linkDestinationOptions = [
+				{ value: href, label: __( 'Media file' ) },
+				{ value: attachmentPage, label: __( 'Attachment page' ) },
+			];
+		}
+
 		const actionButtonStyle = this.props.getStylesFromColorScheme(
 			styles.actionButton,
 			styles.actionButtonDark
 		);
 
 		const isCopyUrlDisabled = isUploadFailed || isUploadInProgress;
-		const dimmedStyle = isCopyUrlDisabled && styles.disabledButton;
-		const finalButtonStyle = Object.assign(
-			{},
-			actionButtonStyle,
-			dimmedStyle
-		);
-
```

Comment:
This is breaking the disabled state style (dimmed 25%)
Probably a merge error?

Reply:
Thanks for spotting. I will check it out. 🙏🏾

Reply:
Resolved in ff743fb